demo/deploy-vault.sh

```diff
@@ -27,9 +27,6 @@ spec:
       labels:
         app: vault
     spec:
-      nodeSelector:
-        kubernetes.io/arch: amd64
-        kubernetes.io/os: linux
```

Comment:
I'd keep the nodeSelector for os but skip arch for the same reason as mysql re: no windows support.